vendor/init.js

```diff
@@ -0,0 +1 @@
+Vue.use(window.vuelidate.default);
```

Comment:
can this be defined in core.js instead of being in a separate file? I think we have other Vue.use declarations.

Reply:
Defined in core.js instead of being in a separete file.